### DISCUSSION TITLE
The app requires a pin

### DISCUSSION THREAD

**Discussion by Xvostillo** [akopachov/flipper-zero_authenticator#132] [Q&A] [answered]
I'm trying to use the application, but when you start the application requires you to enter a certain pin.  The pin on the flipper is not set.  what's my mistake?

**Comment by akopachov** [comment, marked as answer]
When you are running application for the first time it asks you whether you want to setup PIN or not. If you setup it before - application will ask you this PIN every single time you launch application.

Just in case if you would like to reset application to default (the PIN will be reset, but also you will loose all the settings and tokens you setup) [there is a way to do that](https://github.com/akopachov/flipper-zero_authenticator/wiki/How-to-reset-application%3F)

**Reply by Xvostillo** [reply]
thanks, it worked!
